File: pascal/SomarQuatroNumeros.pas
program SomaQuatroNumeros;
var
  num1, num2, num3, num4, soma: integer;
begin
  write('Digite o primeiro número: ');
  readln(num1);

  write('Digite o segundo número: ');
  readln(num2);

  write('Digite o terceiro número: ');
  readln(num3);

  write('Digite o quarto número: ');
  readln(num4);

  soma := num1 + num2 + num3 + num4;
  writeln('A soma é: ', soma);
end.

File: kotlin/SomarQuatroNumeros.kt
fun main() {
    print("Digite o primeiro número: ")
    val num1 = readLine()!!.toInt()

    print("Digite o segundo número: ")
    val num2 = readLine()!!.toInt()

    print("Digite o terceiro número: ")
    val num3 = readLine()!!.toInt()

    print("Digite o quarto número: ")
    val num4 = readLine()!!.toInt()

    val soma = num1 + num2 + num3 + num4
    println("A soma é: $soma")
}